dbt/models/iasworld/schema/iasworld.asmt_all.yml

```diff
@@ -39,7 +39,11 @@ sources:
                     description: class code must be valid
               - res_class_matches_pardat:
                   name: iasworld_asmt_all_class_matches_pardat_class
-                  select_columns_aggregated_with_max: *select-columns
+                  additional_select_columns:
+                    - {"column": "taxyr", "agg_func": "max"}
+                    - {"column": "parid", "agg_func": "max"}
+                    - {"column": "who", "agg_func": "max"}
+                    - {"column": "wen", "agg_func": "max"}
```

Comment:
**question (blocking):** Is this just a compacted way of doing the following?

```yaml
- column: taxyr
  agg_func: max
```

If so, I think I prefer the YAML case even if it's more verbose. If we do want to keep the JSON string we should add a test case for it.

Reply:
Yup, and I switched to the more verbose form for this case and a few other instances in 3109ebac03f62c9bfaca78c28c2151b41d474b65! I'll start expanding these out going forward, I agree that it's easier to read.